src/QMCWaveFunctions/EinsplineSetBuilder_createSPOs.cpp

```diff
@@ -72,6 +75,7 @@ EinsplineSetBuilder::createSPOSetFromXML(xmlNodePtr cur)
   std::string spo_prec("double");
   std::string truncate("no");
   std::string hybrid_rep("no");
+  std::string optimizable("no");
 #if defined(QMC_CUDA)
```

Comment:
Since the orbital rotation scheme doesn't touch the einspline SPO. Optimization switch should be introduced outside, probably in the slater determinant builder.